manifests/init.pp

```diff
@@ -114,10 +114,12 @@
   $prd_web_ui_logger_syslog_errors = true,
   $prd_web_ui_port                 = '3080',
   $prd_web_ui_timeout              = '300',
+  $prd_web_ui_base_url             = undef,
 
   ## JSON API Params
   $prd_json_api_access_log           = 'jsonapi_access.log',
   $prd_json_api_base_url             = 'http://localhost',
+  $prd_json_api_url                  = undef,
```

Comment:
Do you think abount a config like this:

Flapjack:
```
   web:
      port: 3080
      base_url: 'http://localhost:3080/'
```

Apache:
```
ProxyPass /flapjack/ http://localhost:3080/
ProxyPassReverse /flapjack/ http://localhost:3080/
```

This results in http://localhost:3080/ URLs in the content of https://hostname/flapjack/ on our setup using Apache 2.2. on Debian Linux.

The only "workaround" I found was changing the `base_url`. Which should be independent of the port.